### ISSUE TITLE
How use the rule 'last_where' on enumerated? 

### ISSUE THREAD

**Issue by mefilt** [realm/SwiftLint#3492]
How use the rule 'last_where' on enumerated? 
Or how ignore this warning?
<img width="437" alt="image" src="https://user-images.githubusercontent.com/1247906/104779411-2570f900-5790-11eb-85b6-3e2f089d47c1.png">

**Comment by marcelofabri** [comment]
This is what it's being suggested:

```swift
let element = numbers.enumerated.last(where: { $0.element != nil })
```
